dojo/jira_link/helper.py

```diff
@@ -646,6 +647,12 @@ def jira_description(obj, **kwargs):
         kwargs["finding_group"] = obj
 
     description = render_to_string(template, kwargs)
+    defect_dojo_obj_url = get_full_url(obj.get_absolute_url())
+    max_length = getattr(settings, "JIRA_DESCRIPTION_MAX_LENGTH", 32768)
```

Comment:
I was curious where the 32768 value came from. I found this in some JIRA issues where people were requesting no limit for this field:
https://jira.atlassian.com/browse/JRACLOUD-59124

What I find here is that the maximum length is technically 32767. Is our number one too many (assuming the environment variable is unset)?

```suggestion
    max_length = getattr(settings, "JIRA_DESCRIPTION_MAX_LENGTH", 32767)
```